src/controllers/product.controller.js

```diff
@@ -230,4 +222,23 @@ const updateOnadd = async (req, res) => {
   }
 };
 
-export default { CreateCollection, DeleteCollection, addproduct, updateOnadd };
+const searchProducts = async (req, res) => {
+  const products = await productsServices.searchproduct(
```

Comment:
you may wrap all this three parameters into one so that the can be passed in the function as one argument to make it easy for function reusability